src/msw/printdlg.cpp

```diff
@@ -964,6 +984,14 @@ bool wxWindowsPrintDialog::ConvertFromNative( wxPrintDialogData &data )
     {
         data.SetFromPage(pd->lpPageRanges[0].nFromPage);
         data.SetToPage(pd->lpPageRanges[0].nToPage);
+
+        wxVector<wxPrintPageRange>& ranges = data.GetPageRanges();
+        ranges.clear();
+        for (DWORD i = 0; i < pd->nPageRanges; i++) {
```

Comment:
Minor style nitpick:
```suggestion
        for (DWORD i = 0; i < pd->nPageRanges; i++)
        {
```